engine/include/cubos/engine/gizmos/gizmos.hpp

```diff
@@ -90,27 +91,47 @@ namespace cubos::engine
         void drawWireBox(const std::string& id, glm::vec3 corner, glm::vec3 oppositeCorner, float lifespan = 0.0F,
                          Space space = Space::World);
 
+        /// @brief Gets whether the left mouse button was pressed over a gizmo.
+        /// @param id Identifier of the gizmo.
+        /// @return Whether the gizmo was pressed.
+        bool pressed(const std::string& id) const;
+
+        /// @brief Gets whether the mouse button is over a gizmo.
+        /// @param id Identifier of the gizmo.
+        /// @return Whether the mouse button is over a gizmo.
+        bool hover(const std::string& id) const;
+
+        void registerInputGizmo(uint32_t id, bool pressed);
```

Comment:
```suggestion
        void handleInput(uint32_t hovered, bool pressed);
```

Also add missing docs, don't forget to specify that this is called automatically by a system registered by the plugin.